<launch>

	<!-- Launch rosbag -->
	<node pkg="rosbag" type="play" name="player" output="screen" args="--loop --clock /home/ruebenm/workspaces/privacy_ws/src/plane_shaver/bags/my-desk.bag" />

	<!-- Launch AR Tag tracker -->
	<arg name="marker_size" default="4.4" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/camera/depth_registered/points" />
	<arg name="cam_info_topic" default="/camera/rgb/camera_info" />		
	<arg name="output_frame" default="/camera_rgb_optical_frame" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

	<!-- Launch plane redacter node -->
	<node name="redact_plane" pkg="plane_shaver" type="redact_plane" output="screen" />

	<!-- Launch image_view -->
	<node name="image_view" pkg="image_view" type="image_view" >
	      <remap from="/image" to="/camera/rgb/image_filtered" />
	</node>

</launch>